provision/testdata/list_flavors.stdout.txt

```diff
@@ -30,6 +30,8 @@ INFO: kubernetes-1.25:	Kubernetes 1.25
 INFO: RKE2-kubernetes-1.26:	Rancher Kubernetes Engine Government kubernetes version 1.26
 INFO: RKE2-kubernetes-1.25:	Rancher Kubernetes Engine Government kubernetes version 1.25
 INFO: RKE2-kubernetes-1.24:	Rancher Kubernetes Engine Government kubernetes version 1.24
+INFO: RKE-1.5.2-experimental:	Rancher Kubernetes Engine min version 1.5.2
+INFO: RKE-1.4.13-experimental:	Rancher Kubernetes Engine min version 1.4.13
```

Comment:
"experimental" should not be in the name of the flavor, it should be in the description, see L42 below. That way we can just update the description later to remove the "experimental" qualification. The way it stands now, the user will have to change the flavor name from experimental to without it (essentially its a new flavor).